File: Sorting and Searching/linear_search/linear_search.swift
func linear_search(arr: inout [Int], key: Int)->String{
	for index in arr{
		if key == index{
			return "found"
		}
	}
	return "Not Found"
}

var arr = [1,2,3,7,5,6]
print(linear_search(arr: &arr,key: 2))

File: Sorting and Searching/linear_search/linear_search.java
public class linear_search{

    // Function for Linear Search
    public static int linearSearch(int[] arr, int key){
        for(int i=0;i<arr.length;i++){
            if(arr[i] == key){
                return i;
            }
        }
        return -1; 
    }
    public static void main(String a[]){
        // Sample Array
        int[] a1= {10,20,30,40,50,60};
        int key = 30;
        System.out.println(key+" is found at index: " + linearSearch(a1, key)); 
    }
}